src/app/main/component/user/components/profile/users-friends/friend-dashboard/friend-profile-page/friend-profile-dashboard/friend-profile-dashboard.component.scss

```diff
@@ -87,27 +87,49 @@
 }
 
 @media (min-width: 1200px) {
+  .dashboard-container {
+    max-width: 900px;
+    overflow-x: auto;
+  }
+
   .friends-list {
     display: grid;
     grid-template-columns: 417px 417px;
   }
 }
 
+@media (min-width: 1440px) {
```

Comment:
You could use the `responsiveMobileFirst` mixin  from `src/typography/_resp.scss`  instead of directly writing a media query like `@media (min-width)`